web/src/helpers/translation.js

```diff
@@ -1,6 +1,6 @@
 var exports = module.exports = {};
 
-const LOCALES_LIST = ['ar', 'cs', 'da', 'de', 'en', 'es', 'fr', 'hr', 'it', 'ja', 'nl', 'pl', 'pt-br', 'ru', 'sk', 'sv', 'zh-cn', 'zh-hk', 'zh-tw'];
+const LOCALES_LIST = require('../../locales/locales-list');
```

Comment:
CI Error:
ModuleNotFoundError: Module not found: Error: Can't resolve '../../locales/locales-list' in '/home/web/src/helpers'

Reply:
Maybe it's, because it's outside of src?

Reply:
I think that's the reason indeed

Reply:
I know what the issue is🤦

Reply:
The main issue was, that the transpiler can't use the file generated in the other container. I don't know if it also was, because it was outside of src.